scripts/optimize_swarm.py

```diff
@@ -0,0 +1,104 @@
+import subprocess
```

Comment:
please add doc that 
"This file implements a Docker-wrapped detector for HTMcore, which can be used by parameter optimization framework. 
Run: TODO.. "